components/collapsible/collapsible_lazy_show.vue

```diff
@@ -160,6 +158,6 @@ export default {
   .enter-active,
   .leave-active {
     overflow: hidden;
-    transition: height var(--td300) var(--ttf-quint);
+    transition: height var(--td300) var(--ttf-out-quint);
   }
 </style>
```

Comment:
So the real fix is this?
All the other additions you will remove them before merte this PR, right?

Reply:
I was thinking about leaving the story but if it adds no real value I can remove it